TACC/tup-ui
# Tup 376 abstract publiations grants listing

## Overview: For a chosen project, display the project description/abstract, publications, and grants

## Related:

- [TUP-376](https://jira.tacc.utexas.edu/browse/TUP-376)

## Changes:
added 2 tup hooks: usePublications, useGrants
Modified the ProjectDetails page to display descriptions/publication/grants

## Testing:

1. Go to `http://localhost:8000/dashboard/`
2. Click on a project title
3. Check that the abstract field displays the correct description. Compare to [TACC User Portal](https://portal.tacc.utexas.edu/projects-and-allocations).

## UI:
![Screen Shot 2023-01-26 at 12 15 54 PM](https://user-images.githubusercontent.com/35277477/214917047-4a4519b3-387d-4e69-a90d-6b00c69a61c4.png)
![Screen Shot 2023-01-26 at 12 16 01 PM](https://user-images.githubusercontent.com/35277477/214917058-6ea3a300-316f-4d1a-9f85-7a11409a50e4.png)




## Notes:
The current TUP site does not display publications and grants. Hence, this page does not either. 
The links for Edit Abstract, Add Publication, Add Grant do not go anywhere.
